apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -1413,8 +1414,8 @@ object ChatController {
             ((mc is MsgContent.MCImage && file.fileSize <= MAX_IMAGE_SIZE_AUTO_RCV)
                 || (mc is MsgContent.MCVideo && file.fileSize <= MAX_VIDEO_SIZE_AUTO_RCV)
                 || (mc is MsgContent.MCVoice && file.fileSize <= MAX_VOICE_SIZE_AUTO_RCV && file.fileStatus !is CIFileStatus.RcvAccepted))) {
-          // TODO encrypt images and voice
-          withApi { receiveFile(r.user, file.fileId, encrypted = false, auto = true) }
+          val imageOrVoice = mc is MsgContent.MCImage || mc is MsgContent.MCVoice
```

Comment:
TODO - this should encrypt all files but videos, we can use existing method in ChatModel

Reply:
although files won't get to auto receive anyway.

Reply:
and it should also check for protocol